clarity_ext/dilution.py

```diff
@@ -13,18 +13,19 @@ class TransferEndpoint(object):
     """
     # TODO: Handle tube racks
 
-    def __init__(self, analyte):
-        self.sample_name = analyte.name
-        self.well = analyte.well
-        self.container = analyte.container
-        self.concentration = analyte.concentration
-        self.volume = analyte.volume
-        self.requested_concentration = analyte.target_concentration
-        self.requested_volume = analyte.target_volume
+    def __init__(self, aliquot):
+        self.sample_name = aliquot.name
+        self.well = aliquot.well
+        self.container = aliquot.container
+        self.concentration = aliquot.concentration
+        self.volume = aliquot.volume
+        self.requested_concentration = get_and_apply(
+            aliquot.__dict__, "target_concentration", None, float)
```

Comment:
The code would be simpler if the aliquot would just have `target_concentration` of type `float`.